Standards/CCSMB-1.md

```diff
@@ -0,0 +1,14 @@
+# CCSMB-1: Procedures for submitting a standard
+
+This RFC defines the protocol for defining and submitting a standard to the CCSMB.
+
+## File names and formatting
```

Comment:
This should *maybe* be capitalized according to standard rules.

Reply:
```suggestion
## File Naming and Formatting
```